roles/core/dns_server/templates/named.conf.j2

```diff
@@ -74,43 +64,17 @@ zone "." IN {
   file "named.ca";
 };
 
-{# Is the host the master DNS? #}
-{% if dns_master in inventory_all_ipv4_addresses %}
```

Comment:
Beware, users might end with several master dns if they run the role on two management nodes. You need to ensure there is a single dns server for all the cluster.